### PR TITLE
Recommended for you test

### PR DESCRIPTION
**Pull request by joelochlann**
Tests demand for a Recommended For You section on fronts. A dummy section is inserted above Opinion and the user can choose to "turn it on" or not. It's a 404 test: clicking "Yes Please" will give a 404 message and clicking "No Thanks" will remove the section.
### Desktop

picture 267
### Tablet

picture 266
### Mobile

picture 265
### 404 message after clicking "Yes Please" button

picture 268

@katebee

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    mediator,
+    recommendedForYouTemplate,
+    profileIcon,
+    rightArrowIcon,
+    guardianLogo
+) {
+    return function () {
+        this.id = 'RecommendedForYou';
+        this.start = '2016-08-02';
+        this.expiry = '2016-09-24';
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js]: Should this be the same date as the Feature Switch expiry (9/9/2016)?

File: static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                rightArrowIcon: svg(rightArrowIcon, ['i-right']),
+                guardianLogo: svg(guardianLogo),
+                description: description,
+                variant: variant
+            }));
+
+            return fastdom.write(function() {
+                $recommendedForYouSection.insertBefore($opinionSection);
+
+                $('.js-feedback-button-yes', $recommendedForYouSection[0]).each(function(el) {
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js]: `$recommendedForYouSection[0]` is repeated quite a lot, should we just store the ref to the first item? Or would that break line 74 `$recommendedForYouSection.remove();` ?

Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js]: yes it will, since it will no longer be bonzo wrapped set

Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js]: Okay, makes sense 👍

**Pull request merged**